TaskForTraining/TaskForTraining/Pages/HomePage.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.PageObjects;
+
+namespace TaskForTraining.Pages
+{
+    class HomePage
+    {
+        private IWebDriver driver;
+
+        public HomePage(IWebDriver driver)
+        {
+            this.driver = driver;
+            PageFactory.InitElements(driver, this);
+        }
+
+        public const string UName = "Test Auto";
+
+        [FindsBy(How = How.XPath, Using = "//span[@class='uname']")]
+        public IWebElement UserName { get; set; }
```

Comment:
Important! All IWebElements must be private, because IWebElement - element on the page. According to Page Object pattern I must encapsulate all elements on the page and provide external interface as methods, that use those elements.
You always can check your code - if your IWebElemnts are public, then you have invalid code. If you use getter for IWebElement - your code is invalid too. Your tests must contain only methods.